stable/wordpress/README.md

```diff
@@ -111,4 +111,70 @@ See the [Configuration](#configuration) section to configure the PVC or to disab
 
 ## Ingress
 
-This chart provides support for Ingress resource. If you have available an Ingress Controller such as Nginx or Traefik you maybe want to set up `ingress.enabled` to true and choose a `ingress.hostname` for the URL. Then, you should be able to access the installation using that address.
+This chart provides support for Ingress resource. If you have available
+an Ingress Controller such as Nginx or Traefik you maybe want to set up
```

Comment:
> If you have available an Ingress Controller such as Nginx or Traefik you maybe want to set up...

Shouldn't it be something like, "If you have _**an** available Ingress Controller_ such as Nginx or Traefik"?